Form-Controls/index.html

```diff
@@ -14,8 +14,48 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <header> <h1>Product Pick</h1>
+    </header>
+    <main>
+      <div class="wrapper">
+        <form id="product-form">
+        <div>
+        <label for="name">Name</label>
+        <input type="text" id="name" name="name" placeholder="Your name should be 9-20 characters long">
+    </div>
+    <div>
+      <label for="email">Email</label>
+      <input type="email" id="email" name="email" placeholder="Enter your email address">
+    </div>
+    <div>
+      <label for="t-shirtcolor">T-shirt Color</label>
+      <select name="t-shirtcolor" id="t-shirtcolor">
+        <option value="Red">Red</option>
+        <option value="Yellow">Yellow</option>
+        <option value="green">green</option>
+      </select>
+    </div>
+    <div> 
+    <label for="tshirtsize">T-shirt Size</label>
+    <select name="tshirtsize" id="tshirtsize">
+     <option value="XS">XS</option>
+     <option value="S">S</option>
+     <option value="M">M</option>
+     <option value="L">L</option>
+     <option value="XL">XL</option>
+     <option value="XXL">XXL</option>
+    </select>
+    </div>
+  
+  
+    <div>
+      <label for="delivery-date">Please enter the delivery date (YYYY-MM-DD):</label>
+      <input type="date" id="delivery-date" required>
```

Comment:
In the acceptance criteria, it asks for you to constrain the date. What attributes could you use to set a minimum and maximum date range?

Reply:
min and max attributes..?